sbt

```diff
@@ -133,7 +131,9 @@ make_url () {
       0.10.* ) echo "$base/org.scala-tools.sbt/sbt-launch/$version/sbt-launch.jar" ;;
     0.11.[12]) echo "$base/org.scala-tools.sbt/sbt-launch/$version/sbt-launch.jar" ;;
           0.*) echo "$base/org.scala-sbt/sbt-launch/$version/sbt-launch.jar" ;;
-            *) echo "$base/org/scala-sbt/sbt-launch/$version/sbt-launch-${version}.jar" ;;
+    *-[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]T[0-9][0-9][0-9][0-9][0-9][0-9]) # ie "*-yyyymmddThhMMss"
+               echo "$base/org/scala-sbt/sbt-launch/$version/sbt-launch-$version.jar" ;;
+            *) echo "$base/org/scala-sbt/sbt-launch/$version/sbt-launch.jar" ;;
```

Comment:
The use of `sbt-launch-${version}.jar` (or `sbt-launch-$version.jar`) is intentional to verify the checksum.  I assume making this change wasn't intentional.

Reply:
indeed, wrong initial diagnostics - sorry for the noise, I though my draft wouldn't trigger attention!